RELEASE-NOTES.txt

```diff
@@ -4,6 +4,7 @@
 * Deselect post title any time a block is added
 * Fix merging of empty paragraph blocks when added from a post's title
 * Fix loss of center alignment in image captions
+* Prevent keybaord dismissal when switching between caption and text block
```

Comment:
s/keybaord/keyboard 😺

Reply:
🤦‍♂️

Thanks for catching that @hypest ! Fixed now.